modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeEntryServiceTest.java

```diff
@@ -257,6 +280,29 @@ private void _testGetListTypeEntryByExternalReferenceCode(User user)
 		}
 	}
 
+	private void _testGetOrAddIncompleteListTypeEntryPermissions(User user)
+		throws Exception {
+
+		ListTypeEntry listTypeEntry = null;
+
+		try (SafeCloseable safeCloseable =
+				LazyReferencingThreadLocal.setEnabledWithSafeCloseable(true)) {
+
+			_setUser(user);
+
+			listTypeEntry =
+				_listTypeEntryService.getOrAddIncompleteListTypeEntry(
+					user.getUserId(),
+					_listTypeDefinition.getListTypeDefinitionId(),
+					RandomTestUtil.randomString());
+		}
+		finally {
+			if (listTypeEntry != null) {
+				_listTypeEntryLocalService.deleteListTypeEntry(listTypeEntry);
+			}
+		}
```

Comment:
I don't think you need to do this because there is a rule in this class:
```
@ClassRule
@Rule
public static final AggregateTestRule aggregateTestRule =
new LiferayIntegrationTestRule();
```

I advise you to check `LiferayIntegrationTestRule.java` and the Rule responsible for deleting the entities created in the test is `testRules.add(DataGuardTestRule.INSTANCE);`

Run the test locally and make sure that it was deleted, please.